web/src/features/map-controls/MapOptionSelector.tsx

```diff
@@ -15,7 +14,6 @@ export default function MapOptionSelector({
   children,
   isMobile,
 }: MapOptionSelectorProps) {
-  const { __ } = useTranslation();
```

Comment:
since we consider any variable that begins with `_` to be a placeholder/allowed unused we didn't catch these unused hooks before. 

There are a few more that I removed as well.